src/xenia/kernel/xfile.h

```diff
@@ -76,6 +76,7 @@ class X_FILE_DIRECTORY_INFORMATION {
 class XFile : public XObject {
  public:
   static const XObject::Type kObjectType = XObject::Type::File;
+  static constexpr uint16_t kMaximalFileNameLength = 260;
```

Comment:
`FileName` should probably be `Path` or changed to `Name` (as it's already in `XFile`, so you'd be accessing it like `XFile::kMaximalNameLength`).

Reply:
I was thinking about it as it wasn't making much sense, but ignored it 😸. Changed to kMaximalNameLength

Reply:
Thinking about it more, it should most certainly be `Path` since it's the maximum length of the entire thing, not individual name components.

Reply:
Done